packages/web/src/utils/index.js

```diff
@@ -55,3 +56,18 @@ export const hasCustomRenderer = (props = {}) => {
 
 export const isEvent = candidate =>
 	!!(candidate && candidate.stopPropagation && candidate.preventDefault);
+/**
+ * To check if two functions are identical
+ */
+export const isIdentical = (a, b) => {
+	if (a === b) {
+		return true;
+	}
+	if (typeof a === 'function' && typeof b === 'function') {
```

Comment:
Won't the above condition already check type for functions, since we are using `===`. I guess it might never go into this condition, as it will return true from line 64?

Reply:
The first condition is just to cover the case when both have `undefined` or `null` value.
The second condition compares(deeply) the returned value of functions to avoid the infinite loop when someone uses arrow functions.

Reply:
I see in that case we can make the first condition more clear by writing
```
if (!a && !b) return true
```

Reply:
agree